cluster.go

```diff
@@ -514,13 +516,21 @@ retry:
 		return newErrResult(err)
 	}
 	resp = cc.Do(ctx, cmd)
+	if resp.Error() == errConnExpired {
+		goto retry
+	}
 process:
 	switch addr, mode := c.shouldRefreshRetry(resp.Error(), ctx); mode {
 	case RedirectMove:
-		resp = c.redirectOrNew(addr, cc, cmd.Slot(), mode).Do(ctx, cmd)
+		ncc := c.redirectOrNew(addr, cc, cmd.Slot(), mode)
+		resp = ncc.Do(ctx, cmd)
+		if resp.Error() == errConnExpired {
+			goto retry
```

Comment:
Shouldn't this go back to L526?

Reply:
Fix https://github.com/redis/rueidis/pull/833/commits/3623adcf14c62b50c88053c1f64bc5bcef126ca8 . Is it correct?